### DISCUSSION TITLE
Multilanguage applications

### DISCUSSION THREAD

**Discussion by RedBlob** [Budibase/budibase#10675] [Help] [answered]
Hello,

For BB developers, I don't think it's a problem to develop an app using an all-English development system.

For the end user, having a totally English interface is not always an advantage.

![Immagine 2023-05-23 173647](https://github.com/Budibase/budibase/assets/70526244/67ef9912-bae1-48a0-aae9-8bc8c6babfd4)

In our case for example the possibility for the user to be able to choose a local language app ( Romanian, Czech, Polish, Greek or Swedish ) is definitely an advantage that the app could give compared to other apps developed by software developers ( for example those who develop with other platforms guarantee the language switch of the end user ).

Do you have plans to develop this feature ?

Thank You for Help

**Comment by ConorWebb96** [comment, marked as answer]
Hey @RedBlob,

Thank you for your inquiry.

We appreciate your suggestion and recognise the importance of implementing the feature you described. However, our development team currently have a series of prioritised enhancements in progress. As a result, we are unable to provide a specific timeline for the initiation or completion of this particular enhancement.

When we do proceed with its implementation, our aim is to enable language customisation for the apps Budibase creates, catering to the user's preferred language. This is unlikely to be integrated with the builder itself.

I hope this answers your question and is helpful for you.

**Comment by RedBlob** [comment]
Hi @ConorWebb96 ,
Thank You very much for your fast reply.
I think it is important to have a complete set of technical features and database lookup functions  ( and BB is on the right way ! ) ........  but I think it is equally important to have customers who are satisfied with the user experience ( unfortunately in Europe there are many end users who prefer local language apps and also choose the vendor based on this possibility ).

Best Regards and thank you again for your help.
